webapp/channels/src/components/admin_console/admin_settings.tsx

```diff
@@ -79,7 +79,7 @@ export default abstract class AdminSettings <Props extends BaseProps, State exte
         }
     };
 
-    protected handleChange = (id: string, value: boolean) => {
+    protected handleChange = (id: string, value: boolean | EmailSettings['PushNotificationServerType']) => {
```

Comment:
Would you mind actually changing the type of `value` to `unknown`?

I imagine that anyone else who migrates one of these components to TypeScript will run into a similar issue where the type of `value` doesn't suit their use case, so I think it would be easiest to make it more general instead of adding more possible types for it

Reply:
Thanks for your feedback! I have updated the PR with the requested change